static/app/components/tabs/tab.tsx

```diff
@@ -123,6 +123,7 @@ const TabWrap = styled('li', {shouldForwardProp: tabsShouldForwardProp})<{
       &, &:hover {
         color: ${p.theme.subText};
         pointer-events: none;
+        cursor: default;
```

Comment:
@ArthurKnaus is the disabled attribute set on the tab? That should be the correct fix here and we should target that state with the `:disabled` CSS selector

Reply:
The tab is a `li` element. The disabled attribute is set on it, though since it is an `li` you cannot use the `:disabled` selector. I guess is is actually an oversight that the disabled attribute ends up on it.
Will change it to a "no js" style version that selects `[aria-disabled='true']` instead.

Reply:
not sure if we need it though,`react-aria` seems to provide all necessary attributes for it to be accessible 🤔